dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -62,9 +74,15 @@ dom0-rpc-qubes.r5-format-deny:
         qubes.VMShell           *           @anyvm @tag:sd-workstation deny
         qubes.VMShell           *           @tag:sd-workstation @anyvm deny
 
+        qubes.VMExec            *           @anyvm @tag:sd-workstation deny
+        qubes.VMExec            *           @tag:sd-workstation @anyvm deny
+
+        qubes.VMExecGUI         *           @anyvm @tag:sd-workstation deny
+        qubes.VMExecGUI         *           @tag:sd-workstation @anyvm deny
+
```

Comment:
These are denied in the `90-default-policy` as well, but since policies are first-match, I am defining these here out of an abundance of caution just so that user error with .policy files can't affect sdw-tagged vms.